src/components/space/charts/UsersByActionChart.tsx

```diff
@@ -76,12 +77,12 @@ const UsersByActionByChart = (): JSX.Element => {
           <XAxis dataKey="name" tick={{ fontSize: 14 }} />
           <YAxis tick={{ fontSize: 14 }} />
           <Tooltip />
-          {types.map((type, index) => (
+          {types.map((type) => (
             <Bar
               key=""
               dataKey={type}
               stackId="1"
-              fill={COLORS[index % COLORS.length]}
+              fill={ACTION_TRIGGER_TO_COLOR[type] || DEFAULT_ACTION_CELL_COLOR}
```

Comment:
Maybe let's have a function 
```suggestion
              fill={getColorForActionTriggerType(type)}
```